package-lock.json

```diff
@@ -1,8 +1,1340 @@
 {
   "name": "@figma/create-widget",
-  "version": "1.0.3",
```

Comment:
woah is there a reason this is different from the package.json version?

Reply:
why did this file change so much? can we back this out?